src/dao/KegDAO.js

```diff
@@ -1,14 +1,23 @@
 // @flow
 
-import type { Keg } from '../index';
+import type { Keg, KegMutator, KegType, KegTypeProps } from '../index';
 
 import DAO from './DAO';
 import KegTranslator from '../translators/KegTranslator';
 import LoadObject from '../LoadObject';
 import { DAO_ENTITIES } from '../constants';
 import { createFilter } from '../filters';
 
-class KegDAO extends DAO<Keg, Keg> {
+export const KEG_PROPS_BY_TYPE: { [KegType]: KegTypeProps } = {
```

Comment:
I don't know if the name belongs here. It seems more like a presentation detail.

I'm thinking that you should probably store the enum value here.

`size` should probably be `maximumOunces`

Reply:
yeah,`name` is presentation, but maximOunces looks more like data?